src/Admin/SiteHealth.php

```diff
@@ -340,16 +349,28 @@ static function ( $available_service ) {
 	 *
 	 * @since 2.2.1
 	 *
+	 * @param int $threshold Threshold in milliseconds.
+	 *
 	 * @return int Threshold.
 	 */
-	public function get_good_response_time_threshold() {
-		/**
-		 * Filters the threshold below which a response time is considered good.
-		 *
-		 * @since 2.2.1
-		 * @param int $threshold Threshold in milliseconds.
-		 */
-		return (int) apply_filters( 'amp_page_cache_good_response_time_threshold', 600 );
+	public function get_good_response_time_threshold( $threshold = 600 ) {
+		if ( version_compare( get_bloginfo( 'version' ), '6.1', '>=' ) ) {
+			/** @deprecated 2.4.3 Use `site_status_good_response_time_threshold` instead. */
```

Comment:
I think we could just add the `@deprecated` tag to the phpdoc below, and this line can be replaced with just:

```suggestion
			/** This filter is documented in src/Admin/SiteHealth.php */
```

See:

https://github.com/ampproject/amp-wp/blob/d9727b6428969dc8dd38e6abe146cfb6d182faff/includes/admin/functions.php#L105

https://github.com/ampproject/amp-wp/blob/d9727b6428969dc8dd38e6abe146cfb6d182faff/includes/amp-helper-functions.php#L706-L718

Reply:
Could this deprecation tag lead to confusion if the user is on WP < 6.1? Because in that case `site_status_good_response_time_threshold` won't be available.

Reply:
We don't want people to use it anymore though, as they should be updating WordPress to a current version. In any case, they won't get the deprecated notice because of the `if` check. 

In any case, I highly doubt this filter is being used by many sites. So I don't think there will be any confusion as nobody will even notice the tag.